scripts/pipelines/build.cross-toolchain.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+SCRIPT_ROOT=$(realpath $(dirname $BASH_SOURCE))
+
+. "${SCRIPT_ROOT}/build.common.sh" "$@"
+
+echo "INFO: Setting SDKMACHINE to x86_64-mingw32"
+sed -i '/^SDKMACHINE/{s/\"x86_64\"/\"x86_64-mingw32\"/;}' ./conf/local.conf
```

Comment:
Nit: technically don't need the block or the escapes in the sed script.

```suggestion
sed -i '/^SDKMACHINE/s/"x86_64"/"x86_64-mingw32"/' ./conf/local.conf
```

Reply:
Done in v2.